tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Extensions/OrderExtensionsTests.cs

```diff
@@ -90,5 +90,28 @@ public void IsServiceRecipientsSectionComplete_ServiceRecipientsViewedFalse_Retu
             var actual = OrderExtensions.IsServiceRecipientsSectionComplete(order);
             actual.Should().BeFalse();
         }
+
+        [Test]
+        public void IsCatalogueSolutionsSectionComplete_NullOrder_ReturnsFalse()
+        {
+            var actual = OrderExtensions.IsCatalogueSolutionsSectionComplete(null);
+            actual.Should().BeFalse();
+        }
+
+        [Test]
+        public void IsCatalogueSolutionsSectionComplete_ServiceRecipientsViewed_ReturnsTrue()
```

Comment:
Please check the method name... `ServiceRecipientsViewed` should be `CatalogueSolutionsViewed`?